### DISCUSSION TITLE
conda-forge CDN sync not running

### DISCUSSION THREAD

**Discussion by chrisburr** [conda/infra#644] [General]
The conda-forge CDN hasn't been synced in over 6 hours, can someone take a look?

```bash
$ curl -L https://s3.amazonaws.com/conda-static.anaconda.org/conda-forge/last-updated
2022-10-28T02:08:14Z
```

**Comment by chrisburr** [comment]
Duplicate of https://github.com/conda/infra/discussions/643 😅

**Comment by barabo** [comment]
It's back online!  Sorry about the downtime - working on fixing the monitoring that failed us.
